internal/job/job.go

```diff
@@ -74,7 +74,7 @@ func (c *job) Start() {
 			log.Println(err)
 		}
 		if result != nil && len(result) == 2 {
-			ctrl.UploadFile(&result[1])
+			go ctrl.UploadFile(&result[1])
```

Comment:
Esto podríamos hacer que el controller devuelva un error y hacer un wrapping de esto en una función anónima.
```suggestion
			go func(){
			   err := ctrl.UploadFile(&result[1])
			   if err != nil {
			     log.Println(err)
			   }
			}()
```

Reply:
Corregido en b8e8f7b